app/src/main/java/com/it3048/mobiledeviceproject/MeetingAdapter.kt

```diff
@@ -8,20 +8,20 @@ import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 
 class MeetingAdapter(val c:Context, val meetingList:ArrayList<MeetingDAO>):RecyclerView.Adapter<MeetingAdapter.meetingViewHolder>() {
-    inner class meetingViewHolder(val v:View):RecyclerView.ViewHolder(v){
+    inner class MeetingViewHolder(val v:View):RecyclerView.ViewHolder(v){
```

Comment:
Changed the object name to be more in line with what the standard Kotlin conventions are for naming: https://kotlinlang.org/docs/coding-conventions.html#naming-rules